content/tutorials/webrtc/datachannels/en/index.html

```diff
@@ -57,93 +57,92 @@
-
-<p>RTCDataChannel can work in unreliable and unordered mode (analogous to User Datagram Protocol or UDP), reliable and ordered mode (analogous to Transmission Control Protocol or TCP) and partial reliable modes:</p>
-
-<ul>
-  <li><strong>Reliable and ordered mode</strong> guarantees the transmission of messages and also the order in which they are delivered. This takes extra overhead, thus potentially making this mode slower. </li>
-  <li><strong>Unreliable and unordered mode</strong> does not guarantee every message will get to the other side nor what order they get there. This removes the overhead, allowing this mode to work much faster.</li>
-  <li><strong>Partial reliable mode</strong> guarantees the transmission of message under a specific condition (such as a retransmit timeout or a maximum amount of retransmissions). The ordering of messages is also configurable.
-</ul>
-
-<p>Performance for the first two modes is about the same when there are no packet losses. However, in reliable and ordered mode a lost packet will cause other packets to get blocked behind it, and the lost packet might be stale by the time it is retransmitted and arrives. It is, of course, possible to use multiple data channels within the same application, each with their own (un)reliable semantics.</p>
-
-<p>We show below how to configure RTCDataChannel to use reliable and ordered or unreliable and unordered mode.</p>
+<p>Next, you learn how to configure <code>RTCDataChannel</code> to use reliable and ordered or unreliable and unordered mode.</p>
 
 <h2 id="just-show-me-the-action">Configuring data channels</h2>
 
 
-<p>There are several simple demos of RTCDataChannel online:</p>
+<p>There are several simple demos of <code>RTCDataChannel</code> online:</p>
 
 <ul>
-  <li><a href="http://simpl.info/dc">simpl.info/dc</a></li>
-  <li><a href="http://webrtc.github.io/samples/src/content/datachannel/basic/">Transmit text</a></li>
-  <li><a href="http://webrtc.github.io/samples/src/content/datachannel/filetransfer/">Transfer a file</a></li>
-  <li><a href="http://pubnub.github.io/webrtc">pubnub.github.io/webrtc</a> (between two PubNub clients)</li>
+ <li><a href="https://simpl.info/dc">simpl.info <code>RTCDataChannel</code></a></li>
+  <li><a href="https://webrtc.github.io/samples/src/content/datachannel/basic/">WebRTC samples Transmit text</a></li>
+  <li><a href="https://webrtc.github.io/samples/src/content/datachannel/filetransfer/">WebRTC samples Transfer a file</a></li>
 </ul>
 
-<p>In these examples the browser is making a peer connection to itself. It is then creating a data channel and sending the message along the peer connection. Finally, your message appears in the box on the other side of the page!</p>
+<p>In these examples, the browser is making a peer connection to itself. It is then creating a data channel and sending the message along the peer connection. Finally, your message appears in the box on the other side of the page!</p>
```

Comment:
Maybe reword slightly: 

In these examples, the browser makes a peer connection to itself, then creates a data channel and sends a message via the peer connection.

Reply:
Will fix